### ISSUE TITLE
DocumentCard: Focus is not visible on links in High Contrast Mode

### ISSUE THREAD

**Issue by khmakoto** [OfficeDev/office-ui-fabric-react#10665]
### Environment Information

- **Package version(s)**: 7.39.2
- **Browser**: Microsoft Edge

### Please provide a reproduction of the bug in a codepen:

The bug is clearly visible in the [website](https://developer.microsoft.com/en-us/fabric#/controls/web/documentcard). 

#### Actual behavior:

The focus indicator is not visible on links inside `DocumentCards` when in High Contrast Mode:

![DocumentCard](https://user-images.githubusercontent.com/7798177/65901132-66ab8280-e36c-11e9-9f91-0ae9869cecb0.gif)

#### Expected behavior:

The focus indicator is visible on links inside `DocumentCards` when in High Contrast Mode.

#### Notes
Bug 4 in #10646 - DocumentCard A11y Pass

**Comment by khmakoto** [comment]
There seems to be something else going on here. The PPT icons are disappearing in Microsoft Edge after focusing on the links, which makes all the styling look screwed up. This does not repro on Google Chrome.

On Edge:

![DocumentCard](https://user-images.githubusercontent.com/7798177/65920172-28758980-e393-11e9-9530-5e14586a6022.gif)


On Chrome:

![DocumentCard](https://user-images.githubusercontent.com/7798177/65920242-5ce94580-e393-11e9-9258-a5c2e2bd4010.gif)

**Comment by khmakoto** [comment]
After #11306 the focus indicator on the `DocumentCards` themselves is more visible, but the focus indicator on the `Links` is still not visible.
